CODIGO/frmOpciones.frm

```diff
@@ -1600,10 +1586,6 @@ Private Sub HScroll1_Change()
     
     On Error GoTo HScroll1_Change_Err
     
```

Comment:
we need to support volume setting here? why remove VolAmbient = HScroll1.Value?

Reply:
Resolved in next patch